lms/templates/learner_dashboard/course_enroll.underscore

```diff
@@ -5,6 +5,22 @@
     </a>
 <% }else{ %>
     <div class="enrollment-info"><%- gettext('not enrolled') %></div>
+    <% if (run_modes.length > 1){ %>
+        <div class="run-select-container">
+            <label class="sr-only" for="select-<%- course_key %>-run">Select Course Run</label>
+            <select id="select-<%- course_key %>-run" class="run-select">
+                <% _.each (run_modes, function(runMode){ %>
+                    <option value="<%- runMode.run_key %>">
+                        <%= interpolate(
+                            gettext('Starts %(start)s'),
+                            { start: runMode.start_date },
+                            true)
+                        %>
+                    </option>
+                <% }); %>
+            </select>
+        </div>
+    <% } %>
```

Comment:
From the comp (http://wuxtjx.axshare.com/#g=1&p=page_4&c=1), Roderick delivered to us, you are correct that we still display the course start and end date even if there is only 1 run. I'll update this PR according to that assumption. 
Notice in the comp, the dates are shown outside of the course_enroll_view section.